module-code/app/securesocial/core/SecureSocial.scala

```diff
@@ -35,7 +37,7 @@ import scala.concurrent.{ ExecutionContext, Future }
  * if available.
  *
  */
-trait SecureSocial extends Controller with I18nSupport {
+trait SecureSocial extends ControllerHelpers with I18nSupport with PlayBodyParsers {
```

Comment:
The scaladoc says "DO NOT EXTEND this trait in user or library code".
https://www.playframework.com/documentation/2.6.x/api/scala/index.html#play.api.mvc.PlayBodyParsers

Reply:
I see. I misunderstood that this PlayBodyParser is needed for injecting parser or parsers in implicit decls. I will arrange it under the DO NOT EXTEND rules.